layouts/news/rss.xml

```diff
@@ -0,0 +1,32 @@
+<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:content="http://purl.org/rss/1.0/modules/content/">
+  <channel>
+    <title>IPFS Cluster News</title>
+    <link>{{ .Permalink }}</link>
+    <image>
+      <url>{{ "/android-chrome-192x192.png" | absURL }}</url>
+      <title>IPFS Cluster News</title>
+      <link>{{ .Permalink }}</link>
+    </image>
+    <description>Recent news on IPFS Cluster</description>
+    <generator>Hugo -- gohugo.io</generator>{{ with .Site.LanguageCode }}
+    <language>{{.}}</language>{{end}}{{ with .Site.Author.email }}
+    <managingEditor>{{.}}{{ with $.Site.Author.name }} ({{.}}){{end}}</managingEditor>{{end}}{{ with .Site.Author.email }}
+    <webMaster>{{.}}{{ with $.Site.Author.name }} ({{.}}){{end}}</webMaster>{{end}}{{ with .Site.Copyright }}
+    <copyright>{{.}}</copyright>{{end}}{{ if not .Date.IsZero }}
+    <lastBuildDate>{{ .Date.Format "Mon, 02 Jan 2006 15:04:05 -0700" | safeHTML }}</lastBuildDate>{{ end }}
+    {{ with .OutputFormats.Get "RSS" }}
+        {{ printf "<atom:link href=%q rel=\"self\" type=%q />" .Permalink .MediaType | safeHTML }}
+    {{ end }}
+    {{ range first 10 .Pages  }}
```

Comment:
I am missing a sort of some kind. Are these sorted by publish date? Just checking...

Reply:
Yup, default order is sorted by  [Weight > Date > LinkTitle > FilePath](https://gohugo.io/templates/lists/#default-weight-date-linktitle-filepath)
`range first 10` gives 10 latest posts (avoids balooning the feed into hundreds of kilobytes)